book_homework5/book_homework5/Book.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace book_homework5
+{
+    class Book
+    {
+       string Title;
+       int Pages;
+       string Indentificator;
+
+       public Book (string title, int pages, string indentificator)
+        {
+            Title = title;
```

Comment:
If you use default constructor and set in him all properties it not neseccary to create get method for each.
use public definer for it and get info from `Book.Title`
But i think autoproperties will help you with it.